src/topp/FLASHDeconv.cpp

```diff
@@ -491,12 +534,13 @@ class TOPPFLASHDeconv : public TOPPBase
     std::vector<DeconvolvedSpectrum> deconvolved_spectra;
     deconvolved_spectra.reserve(map.size());
 
-    std::vector<DeconvolvedSpectrum> decoy_deconvolved_spectra;
-    decoy_deconvolved_spectra.reserve(map.size() * 3);
+    std::vector<DeconvolvedSpectrum> dummy_deconvolved_spectra;
+    dummy_deconvolved_spectra.reserve(map.size() * 3);
```

Comment:
add comment about magic number if there is a reason why 3 is a good number

Reply:
Yep . added.